components/aside/DotDocumentationAside.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import DotCollectionNav from './components/DotCollectionNav';
+import { DotcmsDocumentation } from '@models/DotcmsDocumentation.model';
+
+const DotDocumentationAside = ({ data }: { data: DotcmsDocumentation }): JSX.Element => {
+    return (
+        <div className="aside-menu-container">
```

Comment:
Use tw in the HTML.

Remove as much as possible the class in the global into their respective components.